autoload/android.vim

```diff
@@ -248,9 +248,10 @@ function! android#setClassPath()
     return
   endif
 
-  if exists(":JCstart")
```

Comment:
@DonnieWest JavaComplete2 does not add target android platform sources path and jar path